### ISSUE TITLE
Cannot parse --timerange when download data

### ISSUE THREAD

**Issue by baont** [freqtrade/freqtrade#4044]
## Describe your environment

  * Operating system: Mac 10.15.6 
  * Python Version: 3.8 (`python -V`)
  * CCXT version: 1.32.7 (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2020.11 (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

I follow this page https://www.freqtrade.io/en/latest/data-download/ to download binance data but freqtrade complained it could not parse the timerange
### Steps to reproduce:

run freqtrade with these args: 
download-data --timerange 20200101 --exchange binance --pairs-file user_data/data/binance/pairs.json --timeframe 3m

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

  ```
 Traceback (most recent call last):
  File "/Users/baonguyen/artisky/freqTrade/ari_trade/freqtrade/main.py", line 37, in main
    return_code = args['func'](args)
  File "/Users/baonguyen/artisky/freqTrade/ari_trade/freqtrade/commands/data_commands.py", line 35, in start_download_data
    timerange = timerange.parse_timerange(config['timerange'])
  File "/Users/baonguyen/artisky/freqTrade/ari_trade/freqtrade/configuration/timerange.py", line 107, in parse_timerange
    raise Exception('Incorrect syntax for timerange "%s"' % text)
Exception: Incorrect syntax for timerange "20200101"
  ```

**Comment by Samaoo** [comment]
The syntax for the timerange argument should be ```--timerange 20200101-``` instead of ```--timerange 20200101```.

**Comment by baont** [comment]
Thanks @Samaoo , i didn't try your fix but if that's the case then the doc needs update

**Comment by Samaoo** [comment]
I will make a pull request to update the docs

**Comment by xmatthias** [comment]
Thanks for updating the Docs!

closed by #4045, which updates the docs to fix this typo.
